src/main/java/org/apache/commons/collections4/CollectionUtils.java

```diff
@@ -1791,6 +1791,65 @@ public static int maxSize(final Collection<? extends Object> coll) {
         return list;
     }
 
+    /**
+     * Removes elements whose index are between startIndex and endIndex in the collection and returns them.
+     * This method modifies the input collections.
+     *
+     * @param <E>  the type of object the {@link Collection} contains
+     * @param input  the collection will be operated, can't be null
+     * @param startIndex  the start index to remove element, can't be less than 0
+     * @param endIndex  the end index to remove, can't be less than startIndex
+     * @return collection of elements that removed from the input collection
+     * @since 4.5
+     */
+    public static <E> Collection<E> removeRange(Collection<E> input, int startIndex, int endIndex) {
```

Comment:
Please use `final` when possible; your IDE might do this for you (Eclipse can for example.)